numpy/core/src/multiarray/conversion_utils.c

```diff
@@ -78,6 +78,29 @@ PyArray_OutputConverter(PyObject *object, PyArrayObject **address)
     }
 }
 
+static npy_intp
+intp_from_scalar(PyObject *ob)
+{
+    /*
+    * Convert the given value to an integer. Then store it and check for
+    * errors.
+    */
+    npy_intp value = PyArray_PyIntAsIntp(ob);
+
+    if (error_converting(value)) {
+        if (PyErr_ExceptionMatches(PyExc_OverflowError))
+            PyErr_SetString(PyExc_ValueError,
+                    "Maximum allowed dimension exceeded");
+        else
```

Comment:
```suggestion
        }
        else {
```
and also other places.  NumPy C-style _never_ leaves out curly brackets.

Reply:
Fixed in 6180fb68b20a967163eadfa47a80c1b526a0afe2